internal/cmd/test.go

```diff
@@ -26,17 +25,27 @@ const (
 	LOCAL_HOST      = "localhost"
 )
 
-type CommonCMDTestCases struct {
-	ExpectedExitCode int
-	ExpectedStdout   string
-	ExpectedStderr   string
+type CMDTestParameters struct {
```

Comment:
I like the old name (except that it should be singular).

The log level affects the output and is therefore part of the test case.